smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChat.java

```diff
@@ -2181,6 +2182,23 @@ private void checkPresenceCode(
                 listener.nicknameChanged(from, mucUser.getItem().getNick());
             }
         }
+        //The room has been destroyed
+        if (mucUser.getDestroy() != null) {
+            Jid alternateJid = null;
```

Comment:
mucUser.getDestroy().getJid() returns a string. I can change that later if you want the parsing done in some other place.